src/Core/Content/Sitemap/Commands/SitemapGenerateCommand.php

```diff
@@ -110,6 +110,8 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
                 $languageIds = $salesChannel->getDomains()->map(function (SalesChannelDomainEntity $salesChannelDomain) {
                     return $salesChannelDomain->getLanguageId();
                 });
+
+                $languageIds = array_unique($languageIds);
```

Comment:
I just had another look on your PR and came across this line. You can't just use a unique filter on the `languageIds` here because you can have the same `languageId` for different salesChannels

Reply:
But it is inside the saleschannel loop so every channel gets generated for all languageIds. This just cuts the duplicated languageIds if you have multiple domains using the same languageId or am i missing something